### PR TITLE
Move Display typeclass to it's own file & modify.

### PR DESCRIPTION
**Pull request by toddburnside**
Modifications from the slack conversation.

### CODE REVIEW

File: modules/core/shared/src/main/scala/lucuma/core/util/Display.scala [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+package lucuma.core.util
+
+/**
+  * Typeclass for things that can be shown in a user interface.
+  * @group Typeclasses
+  */
+trait Display[A]     {
+  def shortName(a: A): String
+  def longName(a:  A): String = shortName(a)
+}
```
Comment (swalker2m) [inline modules/core/shared/src/main/scala/lucuma/core/util/Display.scala:13]: Sorry, late to the party.

Should we add some syntax to permit `a.shortName` and `a.longName` in a future PR?  And an

```scala
object Display {
  def apply[A](implicit ev: Display[A]): Display[A] = ev
}
```

for convenient summons?

Comment (cquiroz) [reply modules/core/shared/src/main/scala/lucuma/core/util/Display.scala:13]: Great idea

Comment (tpolecat) [reply modules/core/shared/src/main/scala/lucuma/core/util/Display.scala:13]: Yes, good idea.

Comment (toddburnside) [reply modules/core/shared/src/main/scala/lucuma/core/util/Display.scala:13]: Guess I was a little quick with the merge button. I'll create another PR with this.

**Pull request merged**